rdrf/rdrf/views/context_views.py

```diff
@@ -128,10 +123,11 @@ class RDRFContextCreateView(View, ContextFormGroupHelperMixin):
     template_name = "rdrf_cdes/rdrf_context.html"
     success_url = reverse_lazy('contextslisting')
 
-    @method_decorator(login_required)
     def get(self, request, registry_code, patient_id, context_form_group_id=None):
+        if not self.allowed(request.user, registry_code, patient_id):
```

Comment:
Did we figure out what RDRFContextEditView and RDRFContextCreateView do? Can we remove them?

Reply:
RDRFContextEditView is being used by RDRFContextCreateView which in turn gets used in RDRFContextLauncherComponent via the get_add_action method of ContextFormGroup model.